lexer.c

```diff
@@ -0,0 +1,75 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   lexer.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: pcharton <pcharton@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/04/03 20:31:36 by pcharton          #+#    #+#             */
+/*   Updated: 2021/04/05 15:04:56 by pcharton         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "header.h"
+#include <stdbool.h>
+
+void	get_command(void)
+{
+	char	*line;
+	int		ret;
+
+	ret = get_next_line(0, &line);
+	if (ret == -1)
+	{
+		puts("gnl error");
+		exit(-1);
+	}
+	else if (ret == 0)
+		exit(0);
+	lexer(line);
+}
+
+int	lexer(char *line)
+{
+	char	*tmp;
+	int		i;
+
+	i = 0;
+	while (*line)
+	{
+		tmp = get_word(&line);
```

Comment:
Get_word returns a char* but this one is just displayed on the screen atm.
I either need to store this char * in a struct or transform it